notebooks/explainable-ai-2-deep-dive/explainable-ai-2-deep-dive.ipynb

```diff
@@ -4,16 +4,22 @@
    "cell_type": "markdown",
```

Comment:
<p>could you please remove this cell output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>